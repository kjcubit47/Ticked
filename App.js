
import React, { useState } from "react";
import { NavigationContainer } from '@react-navigation/native';
import LoginScreen from "Containers/LoginScreen";
import { createNativeStackNavigator } from '@react-navigation/native-stack';
const Stack = createNativeStackNavigator();
import HomeScreen from "Containers/HomeScreen";
import SignUpScreen from "Containers/SignUpScreen";
import ListScreen from "Containers/ListScreen";
import SublistDetailScreen from "Containers/SublistDetailScreen";
import { Provider } from "react-redux";
import store from "Redux/Store";
import { GestureHandlerRootView } from "react-native-gesture-handler";

export default function App() {
  return (
    <GestureHandlerRootView>

      <Provider store={store}>

        <NavigationContainer >
          <Stack.Navigator screenOptions={{ headerShown: false }} initialRouteName="LoginScreen"  >
            <Stack.Screen name="LoginScreen" component={LoginScreen} />
            <Stack.Screen name="HomeScreen" component={HomeScreen} />
            <Stack.Screen name="SignUpScreen" component={SignUpScreen} />
            <Stack.Screen name="ListScreen" component={ListScreen} />
            <Stack.Screen name="SublistDetailScreen" component={SublistDetailScreen} />
          </Stack.Navigator>
        </NavigationContainer>
      </Provider>
    </GestureHandlerRootView>
  );
}

