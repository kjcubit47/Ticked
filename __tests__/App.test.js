import { render } from "@testing-library/react-native";
import App from "../App";

describe("APP", () => {
    test("a", () => {
        const { getByText } = render(<App />);

    })
})