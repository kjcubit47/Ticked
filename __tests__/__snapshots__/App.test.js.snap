// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APP a 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      [
        {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <RNSScreenStack
      style={
        {
          "flex": 1,
        }
      }
    >
      <RNSScreen
        collapsable={false}
        gestureResponseDistance={
          {
            "bottom": -1,
            "end": -1,
            "start": -1,
            "top": -1,
          }
        }
        hasLargeHeader={false}
        nativeBackButtonDismissalEnabled={false}
        onAppear={[Function]}
        onDisappear={[Function]}
        onDismissed={[Function]}
        onGestureCancel={[Function]}
        onHeaderBackButtonClicked={[Function]}
        onNativeDismissCancelled={[Function]}
        onTransitionProgress={[Function]}
        onWillAppear={[Function]}
        onWillDisappear={[Function]}
        replaceAnimation="push"
        sheetAllowedDetents="large"
        sheetCornerRadius={-1}
        sheetExpandsWhenScrolledToEdge={true}
        sheetGrabberVisible={false}
        sheetLargestUndimmedDetent="all"
        stackPresentation="push"
        style={
          {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
        swipeDirection="horizontal"
      >
        <View
          accessibilityElementsHidden={false}
          importantForAccessibility="auto"
          style={
            {
              "flex": 1,
              "flexDirection": "column-reverse",
            }
          }
        >
          <View
            collapsable={false}
            style={
              [
                {
                  "flex": 1,
                },
                {
                  "backgroundColor": "rgb(242, 242, 242)",
                },
                undefined,
              ]
            }
          >
            <View
              style={
                [
                  {
                    "backgroundColor": "#d8dee4",
                    "flex": 1,
                  },
                  undefined,
                ]
              }
            >
              <View
                style={
                  {
                    "backgroundColor": "#6b7999",
                    "height": "6%",
                  }
                }
              />
              <Image
                source={1}
                style={
                  {
                    "position": "absolute",
                    "top": 0,
                  }
                }
              />
              <View
                style={
                  {
                    "bottom": 0,
                    "flex": 1,
                    "justifyContent": "center",
                    "position": "absolute",
                    "width": "100%",
                  }
                }
              >
                <View
                  accessibilityState={
                    {
                      "busy": undefined,
                      "checked": undefined,
                      "disabled": undefined,
                      "expanded": undefined,
                      "selected": undefined,
                    }
                  }
                  accessibilityValue={
                    {
                      "max": undefined,
                      "min": undefined,
                      "now": undefined,
                      "text": undefined,
                    }
                  }
                  accessible={true}
                  collapsable={false}
                  focusable={true}
                  onClick={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    {
                      "backgroundColor": "#6b7999",
                      "borderRadius": 5,
                      "opacity": 1,
                      "padding": 20,
                    }
                  }
                >
                  <View>
                    <Text
                      style={
                        [
                          {
                            "color": "#d8dee4",
                            "fontSize": 24,
                            "textAlign": "center",
                          },
                          [
                            {
                              "color": "#d8dee4",
                              "fontSize": 24,
                              "textAlign": "center",
                            },
                            undefined,
                          ],
                        ]
                      }
                    >
                      Login
                    </Text>
                  </View>
                </View>
                <View
                  accessibilityState={
                    {
                      "busy": undefined,
                      "checked": undefined,
                      "disabled": undefined,
                      "expanded": undefined,
                      "selected": undefined,
                    }
                  }
                  accessibilityValue={
                    {
                      "max": undefined,
                      "min": undefined,
                      "now": undefined,
                      "text": undefined,
                    }
                  }
                  accessible={true}
                  collapsable={false}
                  focusable={true}
                  onClick={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    {
                      "backgroundColor": "#6b7999",
                      "borderRadius": 5,
                      "opacity": 1,
                      "padding": 20,
                    }
                  }
                >
                  <View>
                    <Text
                      style={
                        [
                          {
                            "color": "#d8dee4",
                            "fontSize": 24,
                            "textAlign": "center",
                          },
                          [
                            {
                              "color": "#d8dee4",
                              "fontSize": 24,
                              "textAlign": "center",
                            },
                            undefined,
                          ],
                        ]
                      }
                    >
                      Skip
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
        <RNSScreenStackHeaderConfig
          backButtonInCustomView={false}
          backTitleVisible={true}
          backgroundColor="rgb(255, 255, 255)"
          color="rgb(0, 122, 255)"
          direction="ltr"
          disableBackButtonMenu={false}
          hidden={true}
          hideBackButton={false}
          largeTitleHideShadow={false}
          title="LoginScreen"
          titleColor="rgb(28, 28, 30)"
          topInsetEnabled={false}
          translucent={false}
        />
      </RNSScreen>
    </RNSScreenStack>
  </RNCSafeAreaProvider>
</View>
`;
